src/core/Navigation/SideNavigation/SideNavigation/SideNavigation.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { forwardRef, ReactNode, useState } from 'react';
+import {
+  HtmlButton,
+  HtmlDiv,
+  HtmlNav,
+  HtmlSpan,
+  HtmlUl,
+} from '../../../../reset';
+import styled from 'styled-components';
+import { SuomifiThemeConsumer, SuomifiThemeProp } from '../../../theme';
+import { baseStyles } from './SideNavigation.baseStyles';
+import classnames from 'classnames';
+import { Icon } from '../../../Icon/Icon';
+import {
+  StaticIcon,
+  IllustrativeIconKeys,
+  DoctypeIconKeys,
+} from '../../../StaticIcon/StaticIcon';
+import { getConditionalAriaProp } from '../../../../utils/aria';
+
+export interface SideNavigationProps {
+  /** Use the `<SideNavigationItem>` components as children */
+  children: ReactNode;
+  /** Name for the navigation element. Don't use the word "navigation" since it will be read by screen reader regardless. */
+  'aria-label': string;
+  /** HTML nav element will receive this id */
+  id?: string;
+  /**
+   * Normal or small screen variant
+   * @default normal
+   */
+  variant?: 'default' | 'smallScreen';
+  /**
+   * Whether the menu is initially expanded. Only applies to smallScreen variant.
```

Comment:
Same as above. Maybe "Initial expand status for the menu. Only applies to smallScreen variant".

Reply:
Good point. Changed it!